plugins/web-terminal/README.md

```diff
@@ -1,11 +1,33 @@
-# web-terminal
+# Web terminal plugin for Backstage
 
-Welcome to the web-terminal plugin!
+This plugin provides a frontend for [webterminal proxy](https://github.com/janus-idp/webterminal-proxy). Users first enter their user token from the cluster, and then the plugin setups environment. Once it is set up, it connects to `webterminal-proxy`, which finishes setups and passes data between the frontend plugin and pod.
+This plugin uses [xterm.js](http://xtermjs.org/) to simulate a regular terminal.
 
-_This plugin was created through the Backstage CLI_
+## Requirements
 
-## Getting started
+Before we can install this plugin, we need to fulfill the following requirements:
 
-Your plugin has been added to the example app in this repository, meaning you'll be able to access it by running `yarn start`in the plugin directory.
+1. Deployed `webterminal-proxy`
+2. Installed [Web terminal operator](https://docs.openshift.com/container-platform/4.8/web_console/odc-about-web-terminal.html#odc-installing-web-terminal_odc-about-web-terminal)
+3. Modified permissions in `openshift-terminal` namespace so that users can create `devworkspaces`
```

Comment:
I am wondering if it is better to modify permissions for existing namespace (this namespace is created by `Web terminal operator` when it is installed) or creating new namespace that would be used just by this plugin. WDYT? @tumido

Reply:
How is the namespace accessed by this plugin? For now, this seems to be the only way. We can modify this to fetch [`NAMESPACE`](https://github.com/operate-first/service-catalog/blob/c01da85b74c928c4450f9d376c23650b58684b51/plugins/web-terminal/src/components/TerminalComponent/utils/requests.ts#LL1) from settings instead of being hard-coded.

I think `openshift-terminal` seems to be a sane default. Or we can do the same as OCP terminal web terminal does - if the request fails, we can popup a list of namespaces available to the user so they can choose.

For now, let's go with your current content and I'll file a new feature request issue based on this discussion.

Reply:
Since the https://github.com/operate-first/service-catalog/pull/214 was merged I believe this is no longer relevant so I completely removed this line